<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change</class>
 <widget class="QMainWindow" name="change">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditT">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditScore">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>分数</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAdd">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>添加/修改</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonDel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>修改选课信息时请选择添加/修改或删除，选择删除可不填写分数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNum">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>学号</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>姓名</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>院系</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>班级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>选课</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionquit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionquit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
